src/SWP/Bundle/ContentBundle/Loader/ArticleLoader.php

```diff
@@ -49,6 +50,7 @@ public function load($type, $parameters, $responseType = LoaderInterface::SINGLE
         $dm = $this->serviceContainer->get('doctrine_phpcr.odm.document_manager');
         $configurationPath = $this->serviceContainer->getParameter('kernel.root_dir').'/Resources/meta/article.yml';
         $metadataCache = $this->serviceContainer->get('doctrine_cache.providers.main_cache');
+        $publishWorkflowChecker = $this->serviceContainer->get('cmf_core.publish_workflow.checker');
```

Comment:
Generally bad practice to inject the entire service container into a service. Although I guess you might have done this to avoid a circular reference